daml-assistant/src/DAML/Assistant/Install/Github.hs

```diff
@@ -5,67 +5,26 @@
 
 -- | Discover releases from the digital-asset/daml github.
 module DAML.Assistant.Install.Github
-    ( latestURL
-    , versionURL
+    ( versionURL
+    , getLatestVersion
     ) where
 
 import DAML.Assistant.Types
 import DAML.Assistant.Util
 import Data.Aeson
 import Network.HTTP.Simple
+import Network.HTTP.Client ( redirectCount )
 import Control.Exception.Safe
 import Control.Monad
-import Data.List
 import Data.Either.Extra
 import qualified System.Info
 import qualified Data.Text as T
+import qualified Data.Text.Encoding as T
 
 -- | General git tag. We only care about the tags of the form "v<VERSION>"
 -- where <VERSION> is an SDK version. For example, "v0.11.1".
 newtype Tag = Tag { unTag :: Text } deriving (Eq, Show, FromJSON)
 
--- | Name of github release artifact. We only care about artifacts
--- with the name "daml-sdk-<VERSION>-<OS>.tar.gz" where <VERSION>
--- is an SDK version and <OS> is "linux", "macos", or "win". For
--- example, "daml-sdk-0.11.1-linux.tar.gz".
```

Comment:
All this beautiful documentation, gone...

Reply:
it's dead